ci/docker/runtime_functions.sh

```diff
@@ -337,6 +352,7 @@ build_ubuntu_gpu_cmake() {
         -DUSE_MKLML_MKL=0          \
         -DUSE_MKLDNN=0             \
         -DUSE_DIST_KVSTORE=1       \
+        -DUSE_SIGNAL_HANDLER=ON    \
```

Comment:
Please double check to make sure you still cover all build-functions we have

Reply:
I think I did, problem is the scala build crash.